src/lib/routing-table.js

```diff
@@ -70,29 +70,31 @@ class RoutingTable {
       return undefined
     }
 
-    let bestHop = null
-    let bestValue = null
-    let bestRoute = null
+    let bestHop
     routes.forEach((route, nextHop) => {
-      const value = route.amountAt(sourceAmount)
       // If we have a route but not a curve, pick a route at random
       // and get a remote quote. In the future we may refactor this
       // so that multiple next hop options can be returned, and all
       // can be asked for a quote, but for now, it's just the last.
-      if (value === undefined || !bestValue || value.gt(bestValue)) {
-        bestHop = nextHop
-        bestValue = value && value.toString()
-        bestRoute = route
-      }
+      bestHop = getBetterHop(bestHop, {
+        nextHop,
```

Comment:
getBetterHop takes two arguments, `function getBetterHop (currentHop, otherHop)`.
Here, you pass `bestHop` as the `currentHop`, and `{ nextHop, ... }` as the `otherHop`.
Does that mean (if both arguments of `getBetterHop` are of the same type) that `bestHop` also has a field `nextHop`? `nextHop` comes from `routes.forEach((route, nextHop) => {`, so it's a route.
Now I no longer remember what data type a 'hop' is. Is it a ledger name string? Or a Route object?